understanding/21/reflow.html

```diff
@@ -12,68 +12,245 @@ <h1>Understanding Reflow</h1>
+
+        <p>Reflow applies to both horizontally and vertically written languages. For pages where the primary written language expects vertical scrolling (such as English), users do not expect to scroll back-and-forth horizontally to read that content. Similarly, users reading content in a written language/direction that expects horizontal scrolling (such as traditional Chinese and Japanese) do not have to scroll up-and-down vertically to read that content.</p>
+
+        <div class="note">
+          <p>Reflow does not prohibit web pages from presenting both horizontal and vertical scrollbars for individual sections of content, nor does it disallow the use of bi-directional scrollbars at the page (viewport) level, when the content has been enlarged.</p>
+          
+          <p>However, whenever possible it is in the best interest of the user, to reduce physical and congnivite effort, to limit bi-directional scrolling only to the individual sections of content which necessiate such scrolling, rather than allow the page at large to sroll in two directions. For instance, one might maintain a single scrollbar at page level and confine content which needs bi-directional scrolling (such as a table) into a scrollable container.</p>
+        </div>
+
+        <div class="note" id="note-2dlayout">
+          <p>An important factor in being able to support Reflow is for the user agent to allow users to adjust the size of content within specific windows of content. For instance, the browser's viewport.</p>
+          
+          <p>Technologies such as HTML/CSS, PDF and ePub have methods of reflowing content to adjust to the size of the viewport. When appropriately authored, users can use zooming features, provided by the user agent (browser), to enlarge the content within the viewport. Unless otherwise prevented by author styling, the content of the web page will reflow (wrap) to adjust to the size of the viewport - while the actual user agent, and any UI provided directly by the user agent, remain scaled to the zoom level set by the operating system.</p>
+        </div>
+     </section>
+  
       <section>
+        <h3>Vertically and horizontally scrolling content</h3>
+        <p>Where a vertically scrolling interface has <a href="https://www.w3.org/TR/WCAG/#dfn-section">sections of content</a> which scroll horizontally, those sections would need to meet the 320 CSS pixel vertical scrolling requirement or an exception to pass this criterion. Similarly, in a horizontally scrolling interface that has sections of content which scroll vertically, the sections of content would need to meet the 256 CSS pixel horizontal scrolling requirement, or an exception to the criterion.</p>
```

Comment:
i am thinking this is where clarifying content can be provided to call out that this sort of example: https://codepen.io/Giacomo-Petri/pen/poXqemZ is not a pass - and goes against the intent of reflow, as this content _should_ be able to reflow and scroll vertically.  the 256 height needs to be called out again as being a requirement for content that is meant to be _read_ horizontally, where the language would wrap in "columns" instead of "lines"